aggregate/billing_functions/seqr.py

```diff
@@ -436,14 +437,26 @@ async def generate_proportionate_maps_of_datasets(
 
     logger.info(f'Getting proportionate map for projects: {filtered_projects}')
 
-    analyses, crams = await get_analysis_objects_and_crams_for_seqr_prop_map(
+    # So here
```

Comment:
Not sure what this comment is for

Reply:
Placeholder that I didn't fill in, documented now